src/components/spaces/list_item/index.js

```diff
@@ -3,6 +3,7 @@ import { connect } from 'react-redux';
 import { denormalizedSpaceSelector } from '../denormalized-space-selector.js';
```

Comment:
I figure we'll eventually just remove the list_item, as that was the old way of doing it (now only being used by companies)